packages/terra-pills/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@cerner/terra-pills",
+  "main": "lib/index.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "The terra-pills component provides a way to either display a static text or optionally allow trigger actions to launch popovers with the desired content.",
```

Comment:
should this description be changed ? as we are not implementing the changes for pills displaying only text.

Reply:
Changed cfb6aad5b38a9957f1449aa2f75bbf5d001c3c96